plugins/doc_fragments/onepassword.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2023, Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+class ModuleDocFragment(object):
+    DOCUMENTATION = r'''
+requirements:
+  - See U(https://support.1password.com/command-line/)
+options:
+  _terms:
+    description: Identifier(s) (UUID, name, or subdomain; case-insensitive) of item(s) to retrieve.
+    required: true
```

Comment:
I would keep `_terms` in each individual lookup. (As opposed to other options it is not something that could be shared for example with a module.)